main.nf

```diff
@@ -40,24 +25,18 @@ params.rsem_index   = Workflow.getGenomeAttribute(params, 'rsem')
 params.salmon_index = Workflow.getGenomeAttribute(params, 'salmon')
 
 ////////////////////////////////////////////////////
-/* --         PRINT PARAMETER SUMMARY          -- */
-////////////////////////////////////////////////////
-
-def summary_params = NfcoreSchema.paramsSummaryMap(workflow, params, json_schema)
-log.info NfcoreSchema.paramsSummaryLog(workflow, params, json_schema)
-log.info Workflow.citation(workflow)
-log.info Utils.dashedLine(params.monochrome_logs)
-
-////////////////////////////////////////////////////
-/* --          PARAMETER CHECKS                -- */
+/* --          VALIDATE PARAMETERS             -- */
 ////////////////////////////////////////////////////
 
+def json_schema = "$projectDir/nextflow_schema.json"
```

Comment:
Can we define this in the library as it's always the same? Could accept an optional function argument if we want to allow people to customise it..

Reply:
Wrote a function [here](https://github.com/drpatelh/rnaseq/blob/f1f5e891d4bcfd9c45965857b33b3cc66f2c2ce6/lib/NfcoreSchema.groovy#L16-L19) which does this. Tried various ways but `$projectDir` has to be accessed via evaluation with `workflow.projectDir` so this is the best way I could think of to do this.